example/index.js

```diff
@@ -7,15 +7,37 @@ var options = {
   reporters: [{
     reporter: require('../'),
     args:[
-      { error: '*' },
       {
         auth_token: '1234',
-        room: 'Room',
+        room: 'Devs',
         roomToken: '1234',
-        color: 'red',
         debug: true,
-        formatMessage: function(event) {
-          return event.error.toString() + ' Details: http://example.com?e=something';
+        events: {
+          'error': {
+            '*': {
+              color: 'red',
+              room: 'Emergency Ops',
+              roomToken: '56w34',
```

Comment:
If you dont pass it in via the tag config it'll just use the default above. Could probably use some error checking.